canonical-web-and-design/jaas.ai
# Update run script to latest

## Done

Update run script to latest (this is first step before investigating why all other dependency PRs are failing)

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Check all tests pass and you can spin up the site locally
